map:
  x_size: 40.0
  y_size: 40.0
  z_size: 5.0
  ground_height: -1
  resolution: 0.1
  pub_freq: 1.0
#  frame_id: "map"
  random_seed: 609
  # clear area for start zone or end zone
  clear_area:
    x: [0,0]
    y: [15,-15]
  clear_radius: [1.5,1.5]


ground:
  enable: true
  visual_no_ground: true
  # Default the same as map
#  x_size: 20.0
#  y_size: 20.0
  z_size: 0.0
#  resolution: 0.1

cylinder:
  num: 140 #70
  angle_resolution: 0.1
  height_min: 4.0
  height_max: 4.5
  radius_min: 0.25
  radius_max: 0.25
  fix:
    x: []
    y: []
#  fix:
#    x: [-1,-0.5,-0.2, 0, 1, 2, -0.1]
#    y: [1,1.2,0.8, 1,1,1, -9]

circle:
  num: 0
  height_min: 1.5
  height_max: 4.0
  radius_min: 0.5
  radius_max: 0.7
  thickness_min: 0.1
  thickness_max: 0.1

pillar:
  num: 0
  height_min: 1.5
  height_max: 4.0
  width_min: 0.5
  width_max: 0.7
  length_min: 0.5
  length_max: 0.7

spline:
  num: 80
  height_min: 4.0
  height_max: 4.5
  # The width of the tree branch
  radius_min: 0.2
  radius_max: 0.3
  bending_rate: 0.5
  # The width of the whole tree
  width: 3.0
  sample_t_reso: 0.05
  sample_theta_reso: 0.5
  max_branch_num: 10